setup-tpm2-totp.sh

```diff
@@ -0,0 +1,30 @@
+
+sudo apt -y install build-essential autoconf autoconf-archive automake m4 libtool gcc pkg-config libqrencode-dev pandoc doxygen liboath-dev iproute2 plymouth libplymouth-dev libssl-dev libjson-c-dev libcurl4-openssl-dev
+
+mkdir ./tpm2-totp
+cd tpm2-totp
+
+git clone https://github.com/tpm2-software/tpm2-tss
+cd tpm2-tss
+./bootstrap
+./configure
+make 
+sudo make install
+cd ..
+
+git clone https://github.com/tpm2-software/tpm2-totp
+cd tpm2-totp
+./bootstrap
+./configure
+make
+sudo make install
+
+sudo ldconfig
+
+sudo tpm2-totp --pcrs=0,7 init
```

Comment:
What's the plan for enrollment here? Maybe a wait for input or something to make sure the QR code gets scanned?

Reply:
for now, I'm leaving this script as a separate thing to run manually before `setup-machine.sh`. But yes, when we have it as part of the default build, then yeah we'll have to pause and confirm that the QR code has been recorded.

Reply:
Ah, my bad I  didn't realize this was a separate script.